dist/alert-dialog/alert-dialog.css

```diff
@@ -1,111 +1,123 @@
 :root {
-  --dialog-scrim-color-hide: rgba(17, 24, 32, 0);
-  --dialog-scrim-color-show: rgba(17, 24, 32, 0.7);
-  --dialog-lightbox-max-width: 616px;
+    --dialog-scrim-color-hide: #11182000;
+    --dialog-scrim-color-show: #111820b3;
+    --dialog-lightbox-max-width: 616px;
```

Comment:
I believe prettier was not run on the app. I simply ran prettier which changed the indent.

Reply:
Fixed this

Reply:
Okay, for more context
Dist was in the prettier ignore list. I believe this was due to it changing the format and this might have been picked up on `npm run build`
I changed it so `npm run build` runs prettier on `dist` folder. This is to keep the same formatting across the app.